kolibri/core/assets/src/api_resource.js

```diff
@@ -28,35 +28,106 @@ class Model {
 
     // force IDs to always be strings - this should be changed on the server-side too
     this.attributes[this.resource.idKey] = String(this.attributes[this.resource.idKey]);
```

Comment:
when create new user, we don't know the id yet, here will force a "undefined" string as id.